tm1637.py

```diff
@@ -199,6 +199,34 @@ def temperature(self, num):
             self.write(self.encode_string(string))
         self.write([_SEGMENTS[38], _SEGMENTS[12]], 2)  # degrees C
 
+    def dectemperature(self, num):
```

Comment:
use dec_temperature to follow methods nomenclature standardization of the project

Reply:
will do